sampleapp/templates/django/forms/widgets/password.jinja

```diff
@@ -1,4 +1,4 @@
-<span " x-data="{input: 'password'}">
+<span x-data="{input: 'password'}">
     <input :type="input"
            name="{{ widget.name }}"
            {% if widget.value != None %} value="{{ widget.value|stringformat:'s' }}"{% endif %}
```

Comment:
There's a spelling error in this file as well, although it was there before you touched it - cusor -> cursor